install_cexts.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python
+import os
+import subprocess
+import argparse
+import imp
+
+
+DIST_CEXT = 'kolibri/dist/cext'
```

Comment:
I came to wonder a bit after making my comments: Why should the compiled libraries sit in another target directory? Might they not be together with everything else in `kolibri/dist`?

As per my other comment, I'm sort of leaning towards having the compiled+optional dependencies distributed outside of Kolibri, so it's totally cool to leave it like this.